cmd/copyEnumeratorInit.go

```diff
@@ -521,11 +521,13 @@ func (cca *cookedCopyCmdArgs) makeEscapedRelativePath(source bool, dstIsDir bool
 	return pathEncodeRules(relativePath)
 }
 
-func newFolderPropertyOption(bothFolderAware bool, recursive bool, stripTopDir bool, filters []objectFilter) (common.FolderPropertyOption, string) {
-	if bothFolderAware {
+func newFolderPropertyOption(fromTo common.FromTo, recursive bool, stripTopDir bool, filters []objectFilter) (common.FolderPropertyOption, string) {
+	bothFolderAware := fromTo.AreBothFolderAware()
+	isRemoveFromFolderAware := fromTo == common.EFromTo.FileTrash()
+	if bothFolderAware || isRemoveFromFolderAware {
```

Comment:
Is it possible to consider the trash location (I know we track it as unknown) folder-aware?

It's not like we're using it as a source anywhere.

Reply:
What would be the benefit of that?  Simplifying this code here?  Something else?

Reply:
Simplifying the code here for one. This is a really minor nitpick, so feel free to dismiss.

Reply:
Thanks for clarifying.  Given time constraints, I'm going to leave it as-is, since any change would take a non-zero amount of time ;-)